src/validation/courseIdParamValidator.ts

```diff
@@ -0,0 +1,5 @@
+import Joi from 'joi';
+
+export const courseIdParamValidator = Joi.object({
+  id: Joi.number().guid().required(), 
+});
```

Comment:
This won't run because Joi is not installed, either install it and update all validators to use joi or use express-validator